scripts/xapi-domains.service

```diff
@@ -2,7 +2,8 @@
 Description=Start/stop domains on dom0 start/shutdown
 Requires=proc-xen.mount
 Wants=xapi-init-complete.target
-After=remote-fs.target xapi-init-complete.target
```

Comment:
The ordering should already be there from xapi-init-complete.target -> xapi-wait-init-complete.service -> xapi.service.
Although maybe systemd gets confused about target and service here?

Reply:
Either way it is better to have xapi here explicitly, which we used to have prior to 49d0fbcef5b5712a3352762dae2ae97b4e28bf70#diff-4bbf7bc94efde8928c515b5fece9c92f

Reply:
I'm testing the change, it's not clear to me why is xapi being stopped in parallel with xapi-domains.